src/Builders/SubscriptionBuilder.php

```diff
@@ -524,8 +524,8 @@ protected function throw(Throwable $exception): void
      * Update the subscription's discount free items selection.
      *
      * @param  \AllDressed\Subscription  $subscription
-     * @param  \Illuminate\Support\Collection  $collection
      * @param  \AllDressed\Menu|null  $menu
+     * @param  \Illuminate\Support\Collection  $collection
```

Comment:
Why did you change the position of this in the docs even though the collection is the 2nd parameter? 🤔